doc/modules/cross_validation.rst

```diff
@@ -554,3 +554,20 @@ Cross validation and model selection
 Cross validation iterators can also be used to directly perform model
 selection using Grid Search for the optimal hyperparameters of the
 model. This is the topic of the next section: :ref:`grid_search`.
+
+Cross validation of time series data
+====================================
+
+Time series data is characterised by the correlation between observations 
+that are near in time (*autocorrelation*). However, classical 
+cross-validation techniques such as :class:`KFold`, :class:`ShuffleSplit`,
+and :class:`LeaveOneLabelOut` assume the samples are independent and 
+identically distributed, and would result in unreasonable correlation 
+between training and testing instances (yielding poor estimates of 
+generalisation error) on time series data. Therefore, it is very important 
+to evaluate our model for time series data on the "future" observations 
+least like those that are seen by our model before. To achieve this, we 
+need to adopt the special time series cross-validation which first split 
+the data into :math:`n_splits + 1` folds, then in each iteration 
```

Comment:
math is not going to render this right: the underscore means subscript in TeX. You can either use double-backticks, or change `n_splits` to something like `\texttt{n\_splits}` (not sure if that's sufficient; you can render it and check)